doc/conf.py

```diff
@@ -71,7 +71,7 @@
 # source_encoding = "utf-8-sig"
 
 # Generate the plots for the gallery
-plot_gallery = "True"
+plot_gallery = True
```

Comment:
Does it change anything in practice ? The gallery was already generated.

Reply:
No, but it removes a warning when building the doc with sphinx